packages/typescript-plugin/src/module-loader.ts

```diff
@@ -119,32 +151,22 @@ export function patchModuleLoader(
             return resolved;
         }
 
-        return resolved.map((moduleName, idx) => {
-            const fileName = moduleNames[idx];
-            if (moduleName || !ensureRealSvelteFilePath(fileName).endsWith('.svelte')) {
-                return moduleName;
+        return resolved.map((tsResolvedModule, idx) => {
```

Comment:
Is it correct to adjust the code here? I assume that you updated the code to match the new 5.0 API, but when we are on 4.9 it's the old one, so we shouldn't update this.

Reply:
Just trying to reuse some parts of it but not much is reused so I can revert it if you want.

Reply:
Oops I'm sorry, I didn't realize you refactored something out, all good then